package.json

```diff
@@ -42,7 +43,8 @@
     "s3 streaming upload"
   ],
   "scripts": {
-    "test": "./node_modules/.bin/mocha"
+    "test": "./node_modules/.bin/mocha",
+    "integration-test": "./node_modules/.bin/mocha test/integration"
```

Comment:
we usually have a convention to call this `"test:integration"`

Reply:
Good idea, I'll make it.